src/config.js

```diff
@@ -7,6 +7,7 @@ export const runtimeConfig =
         META_TITLE: window?.env?.META_TITLE,
         META_DESCRIPTION: window?.env?.META_DESCRIPTION,
         META_AUTHOR: window?.env?.META_AUTHOR,
+        META_INDEX_STATUS: window?.env?.META_INDEX_STATUS,
```

Comment:
Looks like we need this variable ~ line 67 too!

Reply:
Great catch. Thanks. Resolved.